src/ado2gh/Services/ReposCsvGeneratorService.cs

```diff
@@ -32,13 +32,29 @@ public virtual async Task<string> Generate(string adoPat)
                         var url = $"https://dev.azure.com/{Uri.EscapeDataString(org)}/{Uri.EscapeDataString(teamProject)}/_git/{Uri.EscapeDataString(repo)}";
                         var pipelineCount = await inspector.GetPipelineCount(org, teamProject, repo);
                         var prCount = await inspector.GetPullRequestCount(org, teamProject, repo);
+                        var lastPushDate = await adoApi.GetLastPushDate(org, teamProject, repo);
+                        var commitsPastYear = await adoApi.GetCommitCountSince(org, teamProject, repo, DateTime.Today.AddYears(-1));
+                        var mostActiveContributor = await GetMostActiveContributor(org, teamProject, repo, adoApi);
 
-                        result.AppendLine($"\"{org}\",\"{teamProject}\",\"{repo}\",\"{url}\",{pipelineCount},{prCount}");
+                        result.AppendLine($"\"{org}\",\"{teamProject}\",\"{repo}\",\"{url}\",{pipelineCount},{prCount},\"{lastPushDate:dd-MMM-yyyy hh:mm tt}\",{commitsPastYear},\"{mostActiveContributor}\"");
                     }
                 }
             }
 
             return result.ToString();
         }
+
+        private async Task<string> GetMostActiveContributor(string org, string teamProject, string repo, AdoApi adoApi)
+        {
+            var pushers = await adoApi.GetPushersSince(org, teamProject, repo, DateTime.Today.AddYears(-1));
+            pushers = pushers.Where(x => !x.Contains("Service"));
```

Comment:
need to add `System.Linq`

Reply:
huh? this code compiles as-is

Reply:
Yeah I also wondered why! I noticed it since CodeQL was complaining about it.